crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -923,7 +923,9 @@ def proposal_update
           @proposal["deployment"][params[:barclamp]] = JSON.parse(params[:proposal_deployment])
           @service_object.save_proposal!(@proposal)
           # validation already happened on save
-          answer = @service_object.proposal_commit(params[:name], validate: false, validate_after_save: false)
+          answer = @service_object.proposal_commit(params[:name],
```

Comment:
Bah OK, you fixed the hound complaints as a separate commit after introducing some new violations :-)  You can see how that causes confusion when reviewing commit by commit.  This is [one of my pet peeves](http://blog.adamspiers.org/2015/03/24/why-and-how-to-correctly-amend-github-pull-requests/#sec-2) which I was explaining to @matelakat earlier :-)